bin/install-cassandra.sh

```diff
@@ -0,0 +1,46 @@
+SAVEDIR=$PWD
+# Change this ROOR_DIR to your preferred instalation directory, and make sure to update cassandraRootDir at SSTableTest.java
+ROOT_DIR="/tmp/"
+
+[ -e $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz ] && rm $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz
+[ -e $ROOT_DIR/cassandra ] && rm -rf $ROOT_DIR/cassandra
+
+# Download Cassandra to $ROOT_DIR
+wget https://downloads.apache.org/cassandra/3.11.6/apache-cassandra-3.11.6-bin.tar.gz -O $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz
+tar xzvf $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz -C $ROOT_DIR
+mv $ROOT_DIR/apache-cassandra-3.11.6 $ROOT_DIR/cassandra
+mkdir $ROOT_DIR/cassandra/logs
+
+# Install Java 8 [Run this if you don't have it]
+# sudo apt-get update
+# printf "Y" | sudo apt-get install openjdk-8-jdk
+
+# Start Cassandra
+$ROOT_DIR/cassandra/bin/cassandra
+
+# Generate 100 flights data (adding unique primary key) from the existing one
```

Comment:
Can't we use autoincrement instead of generating a primary key? For 100 rows it should not take too long.